docs/core-concepts/setup-creds.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: 🔐 Saving Credentials
+sidebar_label: Saving Credentials
+sidebar_position: 6
+unlisted: false
+---
+
+You can setup your on prem credentials directly in Python:
```

Comment:
I would context for _who_ needs to do this: this isn't for all our users, only those that have an on prem setup

Something like:
```
:::info
    This section is only for users with on prem deployments of Fused. If you're using Fused directly on our servers this page doesn't apply to you
:::
```
This is quite exact, but you get the idea